### PR TITLE
feat: add media player subtitles + content: new media pages

### PR DESCRIPTION
**Pull request by Fupete**

### CODE REVIEW

File: src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+            e riguarda appunto, come dire, la messa a punto di una ricerca
+
+            1:04
+            sulle informazioni e le strategie e le risorse
+
+            1:09
+            a supporto delle organizzazioni pubbliche,
+
+            1:13
+            sfruttando in tutto questo quelli che sono gli strumenti e le tecniche di visualizzazione grafica dei dati e delle informazioni. E' un vero e proprio volume
```
Comment (gianvitofanelli) [inline src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml:91]:
```suggestion
            sfruttando in tutto questo quelli che sono gli strumenti e le tecniche di visualizzazione grafica dei dati e delle informazioni. È un vero e proprio volume
```

File: src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+            puntuale,
+
+            4:33
+            che si è svolta appunto con diversi, con un paio di workshop e un paio di incontri
+
+            4:43
+            degli esperti, appunto, del Dipartimento per la trasformazione digitale. Questo è stato interessante soprattutto anche per poter sviluppare un linguaggio comune fra gli studenti e ha permesso di entrare dentro a quello che è il la complessità
+
+            5:04
+            della pubblica amministrazione e di capirne meglio le necessità e le criticità e le problematiche da affrontare. Sembra banale, ma per molti degli studenti, appunto, era la prima volta che si confrontavano con la
```
Comment (gianvitofanelli) [inline src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml:145]:
```suggestion
            della Pubblica Amministrazione e di capirne meglio le necessità e le criticità e le problematiche da affrontare. Sembra banale, ma per molti degli studenti, appunto, era la prima volta che si confrontavano con la
```

File: src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+        - name: MediaPlayer
+          lang: it
+          url: https://www.youtube.com/watch?v=0OzRyE330-I&list=PLd5bJJul8c5qafJpT1RWN3jYIy654Kx5K&index=16
+          poster: /images/community/media-thumbnails/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.jpg
+          subtitles: /files/community/subtitles/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.vtt
+          trascription: |
+            Scarica la [trascrizione in formato ODT](/files/community/subtitles/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.odt). 
+            
+            0:05
+            Sono Gianni Sinni e sono professore di design della comunicazione all'Università Iuav di Venezia, dove mi occupo da tempo di progettazione del design delle informazioni, soprattutto rivolto alla pubblica utilità e a quelli che sono le applicazioni del progetto per quanto riguarda la Pubblica Amministrazione.
```
Comment (gianvitofanelli) [inline src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml:70]:
```suggestion
            Sono Gianni Sinni e sono professore di design della comunicazione all'Università IUAV di Venezia, dove mi occupo da tempo di progettazione del design delle informazioni, soprattutto rivolto alla pubblica utilità e a quelli che sono le applicazioni del progetto per quanto riguarda la Pubblica Amministrazione.
```

File: src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+            sulle informazioni e le strategie e le risorse
+
+            1:09
+            a supporto delle organizzazioni pubbliche,
+
+            1:13
+            sfruttando in tutto questo quelli che sono gli strumenti e le tecniche di visualizzazione grafica dei dati e delle informazioni. E' un vero e proprio volume
+
+            1:27
+            di 416 pagine che è stato sviluppato durante il Laboratorio di design della comunicazione del corso di Laurea magistrale dell'Università Iuav.
```
Comment (gianvitofanelli) [inline src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml:94]:
```suggestion
            di 416 pagine che è stato sviluppato durante il Laboratorio di design della comunicazione del corso di Laurea magistrale dell'Università IUAV.
```

File: src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+            caratterizzanti di una serie di ambiti della pubblica amministrazione nel quale, appunto, sono presenti i servizi digitali.
+
+            2:05
+            Data anche, appunto la complessità del tema, si è limitato la ricerca, diciamo a 5 ambiti, che sono la gestione dei quartieri, l'istruzione, la la sanità,
+
+            2:17
+            la cultura e la mobilità. E all'interno di questi sono stati sviluppati poi tematiche specifiche da parte dei vari gruppi di studenti. L'aspetto importante è che questo Atlante, questa mappatura dei servizi è stata fatta,
+
+            2:35
+            da un'università di progetto che è lo Iuav appunto, come dicevo, in collaborazione con un'amministrazione centrale dello Stato che è il Dipartimento per la trasformazione digitale e Designers Italia. E quindi questo è stato un po' anche come dire, l'elemento caratterizzante di questo progetto, perché è una collaborazione abbastanza rara
```
Comment (gianvitofanelli) [inline src/data/content/community/media/20230203-l-atlante-per-la-trasformazione-digitale-della-pa.yaml:109]:
```suggestion
            da un'università di progetto che è lo IUAV appunto, come dicevo, in collaborazione con un'amministrazione centrale dello Stato che è il Dipartimento per la trasformazione digitale e Designers Italia. E quindi questo è stato un po' anche come dire, l'elemento caratterizzante di questo progetto, perché è una collaborazione abbastanza rara
```

File: src/data/content/community/media/20230215-il-service-design-nella-PA-italiana.yaml [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+            0:30
+            Come mai in questo momento storico il design dei servizi digitali sta assumendo un ruolo così centrale rispetto al settore pubblico? Come sempre in questi casi, la risposta non può essere una sola, sicuramente.
+
+            0:44
+            Le persone hanno nel tempo acquisito sempre più competenze nel mondo del digitale e quindi si rendono sempre più conto che non è tanto la tecnologia essere rilevante per loro, ma quale valore i servizi che vengono mediati dalla tecnologia gli possono portare come li fanno sentire, a che cosa gli danno accesso oppure quale fatica gli evitano. Secondo luogo questo chiaramente viene recepito dalle aziende che si occupano di progettare per il digitale o prodotti digitali o piattaforme digitali.
+
+            1:15
+            Al punto tale che le aziende che sono che integrano i processi di design, in particolare di design, dei servizi all'interno delle loro organizzazioni. Alcune ricerche dicano che arrivano a performare anche due volte meglio rispetto ai benchmark di mercato.
```
Comment (gianvitofanelli) [inline src/data/content/community/media/20230215-il-service-design-nella-PA-italiana.yaml:79]:
```suggestion
            Al punto tale che sono le aziende che integrano i processi di design, in particolare di design dei servizi, all'interno delle loro organizzazioni. Alcune ricerche dicano che arrivano a performare anche due volte meglio rispetto ai benchmark di mercato.
```

File: src/data/content/community/media/20230215-il-service-design-nella-PA-italiana.yaml [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+            1:29
+            Questo non poteva restare fuori dal discorso dei servizi pubblici proprio perché sono i servizi per eccellenza, come tutti hanno diritto e su questo bisogna dire che l'Europa ha fatto sicuramente molto perché a partire dal Digital Compass e dal e Government benchmark e dalle misure degli indicatori del desi, si vede via via sempre più una centralità e un'attenzione legata a come i cittadini possono usufruire dei servizi. Quindi non tanto.
```
Comment (gianvitofanelli) [inline src/data/content/community/media/20230215-il-service-design-nella-PA-italiana.yaml:82]:
```suggestion
            Questo non poteva restare fuori dal discorso dei servizi pubblici proprio perché sono i servizi per eccellenza, come tutti hanno diritto e su questo bisogna dire che l'Europa ha fatto sicuramente molto perché a partire dal Digital Compass e dal e Government benchmark e dalle misure degli indicatori del DESI, si vede via via sempre più una centralità e un'attenzione legata a come i cittadini possono usufruire dei servizi. Quindi non tanto.
```

**Comment by Fupete** [comment]
@gianvitofanelli @danielaiozzo please review all the contents except the subtitles and transcripts textes. I will work on them in the coming days to improve process and result, but in the meantime we have to publish these new media pages.

**Comment by Fupete** [comment]
@astagi @bfabio can you check the subtitles and start from something mechanism code? It seems working to me. 

@gianvitofanelli @danielaiozzo can you review page contents? Forget subtitles and transcripts, we will review them in a dedicated meet next days.

**Review by danielaiozzo — APPROVED** [review]

**Review by astagi — APPROVED** [review]

**Pull request merged**